addons/base_setup/views/res_config_settings_views.xml

```diff
@@ -11,7 +11,7 @@
 
                         <div id="invite_users">
                             <h2>Users</h2>
-                            <div class="row mt16 o_settings_container">
+                            <div class="row mt16 o_settings_container" name="users_setting_container">
```

Comment:
you already have an `id` on the `div` just above, no need to add them on each element

Reply:
@mart-e this is behind the point. This PR doesn't need a review now, I need to know if you'd accept a PR that resolves all the rows without a name or ID. I can then continue to expand the PR (and also resolve this one)

Reply:
This was an example, please try to do so only where it is actually necessary. Don't add names blindly on all the elements, doing an xpath like `//div[@id=invite_users]/div[hasclass('row')][2]` is actually fine. The chance of this breaking in stable version is as small as if you add an id.
So you can go headed on the other apps but keep that in mind to avoid adding too many on them.

Also, try to avoid too long lines, making a one attribute per line is a good practice when it start growing outside of the github window (nobody like horizontal scroll). For instance `<div class="app_settings_block" data-string="CRM" string="CRM" data-key="crm" groups="sales_team.group_sale_manager">` is definitely bad practice ;-)

Reply:
Don't forget my comment about long lines here please